### ISSUE TITLE
Handling of 404's by crawler

### ISSUE THREAD

**Issue by coret** [netwerk-digitaal-erfgoed/register#101]
Currently the crawler keeps checking URL's of (pages with) dataset descriptions indefinitely even if the last crawl couldn't find the specified URL (http 404), see for example https://triplestore.netwerkdigitaalerfgoed.nl/resource?uri=https:%2F%2Fwww.openarch.nl%2Ftest1.jsonld

Suggested behaviour: crawler excluded all URL's with schema:status != 200
(Are HTTP 301/302 followed?)

**Comment by ddeboer** [comment]
But what if the registration URL was _temporarily_ unavailable when being crawled and later works again? Surely we don’t want to stop crawling those URLs.

Some exponential backoff strategy would be best here.

**Comment by coret** [comment]
Good point, but do we already have enough information for an exponential backoff strategy?
May be, a strategy based on the most recent datePosted (=last succesfull crawl).
